app/connectors/NsiConnector.scala

```diff
@@ -60,4 +69,7 @@ class NsiConnector @Inject() (
 
   private val linkAccountsUrl = new URL(baseUrl + getConfig("resources.link"))
   private val checkBalanceUrl = new URL(baseUrl + getConfig("resources.balance"))
+  private val makePaymentUrl  = new URL(baseUrl + getConfig("resources.payment"))
+
+  private val CORRELATION_ID = "Correlation-ID"
```

Comment:
why is this changed from correlation_id to Correlation-ID?

Reply:
Kebab case is the usual convention for HTTP headers. Shall I make it snake case?